### PR TITLE
Fix encoding

### PR DESCRIPTION
**Pull request by hansvancalster**
fixes warning about encoding issue in `read_mow_data()` on Windows.

### CODE REVIEW

**Comment by damianooldoni** [comment]
Nice! I run checks and let you know.

File: R/read_mow_data.R
Comment (damianooldoni) [inline R/read_mow_data.R]: Unfortunately in this way you create a slot called `"wt"`instead of using the value of `wt`. Typical example of use of NSE in R.
Example:
```r
> damiano <- "hans"
> list(damiano = 1)
$damiano
[1] 1
```

I am pushing now a solution.

**Review by damianooldoni — CHANGES_REQUESTED** [review]
Checks are ok, but the last slot has a different name, now. I push my proposed change

File: R/read_mow_data.R
Comment (hansvancalster) [reply R/read_mow_data.R]: Good catch! Thanks for spotting and fixing this 👍

**Comment by hansvancalster** [comment]
Tests run fine, but there are other encoding issues that do not trigger a warning. This happens when I run the example from the vignette:

image

I suggest we solve this later. I'll create an issue for it. For me this branch can be merged, but I wait for your approval.

**Review by damianooldoni — APPROVED** [review]

**Comment by damianooldoni** [comment]
Yes, there are some other problems except the encoding issues you mentioned above, but it has no sense to solve them here. Once this PR is closed and the #101 as well, I will speak about them in a new issue.,

**Comment by damianooldoni** [comment]
You can merge!

**Comment by florisvdh** [comment]
Wait

**Comment by florisvdh** [comment]
> but it has no sense to solve them here

Right, no problem!

Considering remaining encoding issues: @hansvancalster you can have a look at https://github.com/inbo/inborutils/blob/master/R/convertdf_enc.R.

**Comment by florisvdh** [comment]
> I will speak about them in a new issue.,

:+1:

**Comment by hansvancalster** [comment]
> Wait

@florisvdh I already tried to pipe the result to `inborutils::convertdf_enc()`, but it didn't solve the problem.

**Comment by florisvdh** [comment]
OK

**Comment by damianooldoni** [comment]
The typical problems of Windows, I dare to say.

**Comment by florisvdh** [comment]
> The typical problems of Windows, I dare to say.

Well, they're making progress though. Since 2019 it is possible to use UTF-8 system-wide on Win 10, though I think it's still considered 'beta'. It can cause conflicts with Windows software which does not expect UTF-8.

It seems R is going to take advantage of this feature in the future:
- [UTF-8 Support on Windows](https://developer.r-project.org/Blog/public/2020/05/02/utf-8-support-on-windows/index.html)
- [Windows/UTF-8 Build of R and CRAN Packages](https://developer.r-project.org/Blog/public/2020/07/30/windows/utf-8-build-of-r-and-cran-packages/index.html)

**Comment by ElsLommelen** [comment]
However, as long as we don't all have Win 10, it would be nice if package developers would take these differences in encoding into account and also test the code on all three OS before submitting code, certainly when encodings are set explicitly...

**Pull request merged**
